app/src/main/java/com/gh4a/activities/IssueActivity.java

```diff
@@ -113,17 +138,33 @@ protected void onResultReady(Boolean result) {
     };
 
     @Override
+    protected String getActivityTitle() {
+        ActionBar actionBar = getSupportActionBar();
+        if (actionBar == null || actionBar.getTitle() == null) {
+            return null;
+        }
+        return actionBar.getTitle().toString();
+    }
+
+    @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         setContentView(R.layout.frame_layout);
         setContentShown(false);
 
         ActionBar actionBar = getSupportActionBar();
-        actionBar.setTitle(getString(R.string.issue) + " #" + mIssueNumber);
+        String title = getString(R.string.issue) + " #" + mIssueNumber;
+        actionBar.setTitle(title);
         actionBar.setSubtitle(mRepoOwner + "/" + mRepoName);
         actionBar.setDisplayHomeAsUpEnabled(true);
 
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            // TODO: Better handle activity titles
```

Comment:
We might want to add a method for initializing this inside of BaseActivity after intent extras are read. It would then use getActivityTitle() to setup TaskDescription. But I'm not sure if this will be needed in all the other activities that won't be launched in separate tasks...